timesketch/api/v1/resources/sketch.py

```diff
@@ -110,14 +113,49 @@ def post(self):
 class SketchResource(resources.ResourceMixin, Resource):
     """Resource to get a sketch."""
 
+    def _get_sketch_for_admin(self, sketch):
+        """Missing a docstring."""
```

Comment:
It is indeed missing a docstring :)

Reply:
it is really missing it... that is so true

Reply:
done